### PR TITLE
Preview release 2.1 post

### PR DESCRIPTION
**Pull request by stefonarch**

### CODE REVIEW

**Comment by tsujan** [comment]
I'll make the release announcement based on it. But making an announcement can be done only after releasing all sources — as before.

**Comment by stefonarch** [comment]
It may not be complete...
Anything to correct?

File: _posts/2024-09-20-release 2.1-preview.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+Here's a overview of the coming LXQt release 2.1 due in mid November as usual. The main target is the experimental **Wayland session** but also some other improvements will be included.
+
+#### Wayland Session
+
+A new package [lxqt-wayland-session](https://github.com/lxqt/lxqt-wayland-session) will be available to install - without it there won't be seen any changes, following the "modular" philosophy of LXQt: don't bother users with unwanted or unused things. If installed "Session Settings" will look like this:
+
+![Session Settings](../../../../../images/posts/waylandsettings.png)
+
+As Wayland is quite different to X11 some features are not available and some settings are handled by the compositor and not anymore by LXQt, so you may see some messages like that:
```
Comment (tsujan) [inline _posts/2024-09-20-release 2.1-preview.md:18]: As Wayland is quite different **from** X11,..., so you may see some messages like **this**:

File: _posts/2024-09-20-release 2.1-preview.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+![unsupported settings](../../../../../images/posts/unsupported.png)
+
+Similar to the window manager under X11 users can choose a Wayland compositor - any wlroots-compatible compositor should work, in addition to kwin-wayland. Hyprland, Labwc, kwin, Niri, Sway, River and Wayfire will be supported out of the box providing a basic configuration file for each of them. Using different screen lockers under Wayland including `swaylock`, `waylock` and `hyprlock` is supported too.
```
Comment (tsujan) [inline _posts/2024-09-20-release 2.1-preview.md:22]: Users can choose a Wayland compositor, as they chose a window manager under X11 - ...

File: _posts/2024-09-20-release 2.1-preview.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+At the moment there are 3 dedicated panel backends, xcb (X11), general wlroots  and kwin_wayland - using the latter many features of the panel as moving windows to workspaces, workspace switcher, urgency hints, show desktop will be enabled under Wayland. Other backends can be created and added in future, enabling similar features for other compositors like sending to scratchpad instead of minimizing windows, workspace pagers and else. PRs are welcome.
```
Comment (tsujan) [inline _posts/2024-09-20-release 2.1-preview.md:24]: Currently there are 3 dedicated panel backends, xcb (X11), general wlroots and kwin_wayland - with kwin_wayland supporting several features like moving windows to workspaces, workspace switcher, urgency hints, and showing desktop.

File: _posts/2024-09-20-release 2.1-preview.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+As some applications will not work under Wayland (e.g. redshift, some clipboard manager exc) their auto starting can be limited to X11 only, while on the other side every compositor can start Wayland-only applications like `kanshi` or `xdg-desktop-portal-wlr`.
```
Comment (tsujan) [inline _posts/2024-09-20-release 2.1-preview.md:26]: ... while, on the other hand, every compositor can start...

File: _posts/2024-09-20-release 2.1-preview.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+Other fixes for Wayland have been applied overall, including the desktop window of `PCManFM-Qt` which has also been improved for multi monitor setups.
```
Comment (tsujan) [inline _posts/2024-09-20-release 2.1-preview.md:28]: There are also general fixes and workarounds for Wayland; for example, the desktop window of `PCManFM-Qt` has been improved for use with multi monitor setups.

File: _posts/2024-09-20-release 2.1-preview.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+For further details please consult the [Wayland Wiki](https://github.com/lxqt/lxqt/wiki/ConfigWaylandSettings).
+
+The upcoming LXQt 2.1 release can already be tested installing from source, preferably using the AUR packages - as more testing feedback we get the better!
```
Comment (tsujan) [inline _posts/2024-09-20-release 2.1-preview.md:32]: The upcoming LXQt 2.1 release can already be tested by installing from sources or, preferably, by making distro-specific packages and installing them (like AUR packages in Arch). The more testing feedback we get, the sooner we could find and fix probable problems!

**Comment by tsujan** [comment]
> Anything to correct?

I made some suggestions above.

**Pull request merged**